dali/test/python/operator_1/test_arithmetic_ops.py

```diff
@@ -863,6 +864,118 @@ def get_sh(arg_idx):
                 yield check_ternary_op, kinds, types_in, op, get_sh, op_desc
 
 
+def generate_layout_broadcasting_cases():
+    rng = np.random.default_rng(4242)
+
+    def get_input_dev(num_inputs):
+        placement = rng.choice(["cpu", "gpu", "non_uniform"])
+        if placement != "non_uniform":
+            return (placement, ) * num_inputs
+        placement = [rng.choice(["cpu", "gpu"]) for _ in range(num_inputs - 1)]
+        placement.append("gpu" if placement[-1] == "cpu" else "cpu")
+        return tuple(placement)
+
+    def get_input_types(num_inputs, integral_only):
+        types = (np.int32, np.uint8)
+        if not integral_only:
+            types += (np.float32, )
+        return tuple(rng.choice(types, size=(num_inputs, )))
+
+    bin_layouts = [
```

Comment:
An explanation would be nice.
```suggestion
    # The input layouts and the expected output layout.
    # A number N denotes an ND tensor without a layout.
    # `Exception` means that applying an operator with the arguments with given layouts should raise an error.
    bin_layouts = [
```

Reply:
done